spec/lib/patches/db/pg_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'pg'
```

Comment:
This breaks the `SQLPatches` spec, because it'll only patch the `pg`-related code. @SamSaffron can you advise?